python/triton/ops/matmul.py

```diff
@@ -144,7 +145,7 @@ class _matmul(torch.autograd.Function):
     _locks = {}
 
     @staticmethod
-    def _call(a, b, dot_out_dtype, allow_tf32, fp8_fast_accum):
+    def _call(a, b, dot_out_dtype, allow_tf32, fp8_fast_accum, c_dtype):
```

Comment:
rename `dot_out_type` to `accumulation_type`

Reply:
I will use acc_type